modules/net-ilb/variables.tf

```diff
@@ -27,7 +27,7 @@ variable "backend_service_config" {
     connection_tracking = optional(object({
       idle_timeout_sec          = optional(number)
       persist_conn_on_unhealthy = optional(string)
-      track_per_session         = optional(bool)
+      tracking_mode             = optional(string)
```

Comment:
if the choice is just between two values, I would still use a boolean as a string requires figuring out the actual value needed, and is prone to typos.

Reply:
You are right! I realigned back and fixed the issue with boolean, now I assign PER_SESSION and PER_CONNECTION accordingly, when null it defaults to PER_CONNECTION

Reply:
awesome, thanks Simone!